xcp_d/workflows/restingstate.py

```diff
@@ -230,10 +162,52 @@ def init_alff_wf(
         name="ds_alff_plot",
         run_without_submitting=False,
     )
+    workflow.connect([(plot_alff, ds_alff_plot, [("output_path", "in_file")])])
 
-    # fmt:off
-    workflow.connect([(alff_plot, ds_alff_plot, [("output_path", "in_file")])])
-    # fmt:on
+    if smoothing:
+        workflow.__desc__ += " ALFF was also calculated from the smoothed, denoised BOLD data."
+
+        compute_smoothed_alff = pe.Node(
+            ComputeALFF(TR=TR, low_pass=low_pass, high_pass=high_pass),
+            mem_gb=mem_gb["resampled"],
```

Comment:
I was getting `TypeError: '>' not supported between instances of 'dict' and 'float'` because it was originally setting `mem_gb` as the `mem_gb` dictionary instead of the appropriate value.

Hunting down the problem was tough.